### PR TITLE
Import the physical code from capture to tikiClient

### LINKED ISSUE #53: Verify the take a photo of receipt code
Verify the take a photo receipt is working and remove all mock code.

Success:
- [x] Take a picture and save with success

### LINKED ISSUE #58: Import the physical code from capture to tikiClient
The tikiClient was a project who has made start zero and importing only what necessary at the moment, now the physical scan scan is necessary on the project. The logic is already implemented on capture project, is necessary import and test the code inside the tikiClient project.

The reference code is:
```
public class Physical {
    
    private static var scanCallback: PhysicalScanCallbacks? = nil
    
    /// Initiates a receipt scanning process using the device's camera.
    func scan(scanCallback: PhysicalScanCallbacks) {
        Physical.scanCallback = scanCallback
        
        DispatchQueue.main.async {
            let scanResultsDelegate = UIApplication.shared.windows.first!.rootViewController!
            let mediaType = AVMediaType.video
            let authStatus = AVCaptureDevice.authorizationStatus(for: mediaType)

            
            if authStatus == .authorized {

            } else {
                AVCaptureDevice.requestAccess(for: mediaType) { granted in
                    if granted {
                        DispatchQueue.main.async {

                        }
                    } else {
                        scanCallback.onError(NSError(domain: "No camera access", code: -1))
                        Physical.scanCallback = nil
                    }
                }
            }
        }
    }
```

The TikiClient already have a method to implement this:

https://github.com/tiki/publish-client-ios/blob/main/TikiClient/Classes/Capture/CaptureService.swift#L9C1-L11C6

To adapt the result we should add the capture photo code, like in this sample
https://developer.apple.com/tutorials/sample-apps/capturingphotos-captureandsave

Success:
- [x] The capture receipt is able to take a receipt photo and return it as Data

### PR DESCRIPTION
**Pull request by JesseMonteiro**
close #58 
close #53

### CODE REVIEW

File: Example/TikiClient/URLShemeExample.swift
Comment (ricardobrg) [inline Example/TikiClient/URLShemeExample.swift:1]: fix filename `URLScheme...`

File: Physical/Camera.swift
Comment (ricardobrg) [inline Physical/Camera.swift:1]: The client lib should not have any UI. The UI should be in the Example app and it calls the function in the client lib that:
- check/ask the permission
- open the camera
- return the UIImage

File: TikiClient/Classes/Util/Coordinator.swift
Comment (ricardobrg) [inline TikiClient/Classes/Util/Coordinator.swift:1]: This should be in the Example app. No UI in the client lib.

**Review by ricardobrg — CHANGES_REQUESTED** [review]

**Comment by JesseMonteiro** [comment]
Issues solved

File: Physical/ImagePicker.swift
Comment (ricardobrg) [inline Physical/ImagePicker.swift:1]: move this code to TikiClient/Classes/Capture

File: TikiClient/Classes/Capture/CaptureService.swift
Comment (ricardobrg) [inline TikiClient/Classes/Capture/CaptureService.swift:1]: create a CaptureScan class and move this code into it. With this, the CaptureService doesn't need to extend all these classas. The scan function call the CaptureScan class.

File: TikiClient/Classes/Capture/PublishingStatusEnum.swift
Comment (ricardobrg) [inline TikiClient/Classes/Capture/PublishingStatusEnum.swift:1]: this file can be removed, we will not use it now

**Review by ricardobrg — CHANGES_REQUESTED** [review]

File: Physical/ImagePicker.swift
Comment (JesseMonteiro) [reply Physical/ImagePicker.swift:1]: solve

**Pull request merged**